app/flows/check_uk_visa_flow/outcomes/outcome_marriage_electronic_visa_waiver.erb

```diff
@@ -11,5 +11,9 @@
   - settled in the UK
   - someone with humanitarian protection, for example if they have refugee status
 
-  ^If you want to convert a civil partnership into a marriage, you'll need to apply for an [electronic visa waiver (EVW)](/get-electronic-visa-waiver) or a [Standard Visitor visa](/standard-visitor).
+  If you want to convert a civil partnership into a marriage, you'll need to apply for an [electronic visa waiver (EVW)](/get-electronic-visa-waiver) or a [Standard Visitor visa](/standard-visitor).
+
+  <%if calculator.passport_country_requires_electronic_travel_authorisation?  %>
+    <%= render partial: 'electronic_travel_authorisation_advice', locals: {calculator: calculator} %>
```

Comment:
I don't see the calculator local variable being used in `electronic_travel_authorisation_advice.erb`?